### ISSUE TITLE
Dokumentation how to loop the proxy into the connection 

### ISSUE THREAD

**Issue by s-allius** [s-allius/tsun-gen3-proxy#1]
It's unclear how the proxy can be looped into the connection between the inverter and the TSUN cloud.

Some examples would help

**Comment by morkmorks** [comment]
Indeed- Is there an easy way of setting the single DNS entry in the settings of the commonly used FRITZ!Box (w/o using Pihole)?

**Comment by s-allius** [comment]
I'm sorry, but my knowledge is, that you can't add DNS entries on a Fritz!Box directly. So you must use a local DNS Server, like Phiole, for that.
If you have a newer GEN3Plus Inverter, you can configure the connection to the proxy on the Web-Interface of the inverter. The newer inverters have a serial number starting with 'Y17'. Which kind of inverter do you have?

**Comment by morkmorks** [comment]
Hi Stefan

 

Thank you so much for the speedy response.

I have an R17…..-inverter – I bought it last year, so no web-Interface.

 

I will try on the Pihole on the weekend – let’s see!

 

Cool that you care about this stuff!

 

Cheers

 

mm

 

 

Von: Stefan Allius ***@***.***> 
Gesendet: Dienstag, 2. April 2024 18:37
An: s-allius/tsun-gen3-proxy ***@***.***>
Cc: morkmorks ***@***.***>; Comment ***@***.***>
Betreff: Re: [s-allius/tsun-gen3-proxy] Dokumentation how to loop the proxy into the connection (Issue #1)

 

I'm sorry, but my knowledge is, that you can't add DNS entries on a Fritz!Box directly. So you must use a local DNS Server, like Phiole, for that.
If you have a newer GEN3Plus Inverter, you can configure the connection to the proxy on the Web-Interface of the inverter. The newer inverters have a serial number starting with 'Y17'. Which kind of inverter do you have?

**Comment by ryanrudak** [comment]
I did it like that way:
 
	 - enter the tsun-gen3-proxy (IP address or DNS name) as the optional server on the /config_hide.html page of the microinverter
	 - Internet access for the IP address of the mircoinverter is prohibited in the Fritz!Box
 
Since the first server is then no longer accessible, the optional server is used, in this case the tsun-gen3-proxy.
 
This works great for my.

**Comment by jo-mueller** [comment]
Late to the party but I think I'm struggling at the same step.

> If you have a newer GEN3Plus Inverter, you can configure the connection to the proxy on the Web-Interface of the inverter. The newer inverters have a serial number starting with 'Y17'. Which kind of inverter do you have?

Mine starts with Y47 which I believe is indicative of it being of a newer generation. As an interface, I use the TSUN smart app - is that what's meant by the web interface? 🤔
